xgcm/grid.py

```diff
@@ -109,6 +110,15 @@ def __init__(
             boundary kwarg when calling specific methods.
         fill_value : {float}, optional
             The value to use in the boundary condition when `boundary='fill'`.
+        direction : str, optional
+            The direction in which axes are defined:
+            * None: Do not define a specific direction; defaults to same
+              behaviour as 'increasing'
+            * 'increasing': coordinate increases with increasing index, e.g. 
+              latitude increases with increasing y-index.
+            * 'decreasing': coordinate decreases with increasing index, e.g. 
+              depth decreases with increasing z-index.
```

Comment:
I know the naming has already been discussed a bit. I'll give my opinion.

This seems a little complicated and verbose for something that is essentially a binary value. Either the axis is increasing upward or not. So I would weakly favor the `positive_upward=True` convention.